client/src/pages/Badrequest400.jsx

```diff
@@ -0,0 +1,12 @@
+import React, { Fragment } from "react";
+import { Typography } from "@material-ui/core";
+
+export default function Badrequest400() {
+  return (
+    <Fragment>
+      <Typography variant="h2" gutterBottom>
+        400 - Bad Request!
+      </Typography>
+    </Fragment>
+  );
+}
```

Comment:
ok, I'll add it in the next PR

Reply:
Oh, for this part, I remembered that for the signup and login page, if there were some errors, I have added the Snackbar. I add this page is just in case whether there would be any other bad request. I am not sure whether I have thought too much.